pyccl/tracers.py

```diff
@@ -577,16 +564,33 @@ def __del__(self):
                 lib.cl_tracer_t_free(t)
 
 
-def NumberCountsTracer(cosmo, has_rsd, dndz, bias=None,
-                       mag_bias=None, n_samples=256):
+class NzTracer(Tracer):
+    """Specific for tracers with an internal `_dndz` redshift
+    distribution interpolator. Currently only used for
+    :class:`NumberCountsTracer` and :class:`WeakLensingTracer`.
```

Comment:
I'd remove those 2 lines, because once we implement a 3rd tracer, we will 99.9% forget to update this docstring.